quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/PropertyMapper.java

```diff
@@ -525,6 +525,7 @@ public static boolean isEnvOption(ConfigValue configValue) {
     }
 
     void validateExpectedValues(ConfigValue configValue, String v) {
+        v = v.toLowerCase();
```

Comment:
I don't think this generally holds. Other usage of enums, such as proxy-headers, are still case-sensitive.